pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -446,13 +442,21 @@ public void publishMessage(ByteBuf headersAndPayload, PublishContext publishCont
     }
 
     public void updateSubscribeRateLimiter() {
```

Comment:
Does this method need to be synchronized?

Reply:
+1

All other changes to `subscribeRateLimiter ` have already added synchronized lock.

This is a question left over from the last PR #14890